Content.Client/Implants/UI/ImplanterStatusControl.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Content.Client.Message;
+using Content.Client.Stylesheets;
+using Content.Shared.Implants.Components;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.Containers;
+using Robust.Shared.Timing;
+
+namespace Content.Client.Implants.UI;
+
+public sealed class ImplanterStatusControl : Control
+{
+    private readonly ImplanterComponent _parent;
```

Comment:
Does ItemStatusCollectMessage get refreshed if the component is deleted? Just wondering how the lifetime stuff is handled.

Reply:
This is what it looks like when ImplanterComponent is deleted:

![image](https://user-images.githubusercontent.com/54602815/201966869-86ebcf11-2867-459e-87f2-b6077f02323e.png)

And with it as normal:

![image](https://user-images.githubusercontent.com/54602815/201966964-b0c26076-dc03-405c-b188-7f8f41bf7bc8.png)

Reply:
Is that with it added back.